parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -227,18 +231,14 @@ installContainerd() {
     if [[ "$CURRENT_VERSION" == "${CONTAINERD_VERSION}" ]]; then
         echo "containerd is already installed, skipping install"
     else
-        CONTAINERD_TGZ_TMP="cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz"
-        rm -Rf /usr/bin/containerd
-        rm -Rf /var/lib/docker/containerd
-        rm -Rf /run/docker/containerd
-        if [[ ! -f "$CONTAINERD_DOWNLOADS_DIR/${CONTAINERD_TGZ_TMP}" ]]; then
-            downloadContainerd
-        fi
-        tar -xzf "$CONTAINERD_DOWNLOADS_DIR/$CONTAINERD_TGZ_TMP" -C /
-        sed -i '/\[Service\]/a ExecStartPost=\/sbin\/iptables -P FORWARD ACCEPT -w' /etc/systemd/system/containerd.service
-        echo "Successfully installed cri-containerd..."
+        removeContainerd
```

Comment:
Shall I infer that this is a "best practice, just-in-case" type operation? In other words, the most deterministic way to install a particular version of the `moby-containerd` is to ensure no pre-existing `moby-containerd` and `moby-runc` packages aren't present on the OS already?

Also, is `moby-runc` installed as a dependency of `moby-containerd`? Or is it just a conflicting package and so we need to ensure it's removed before we install `moby-containerd`?

Reply:
I copied this pattern from `installMoby` is all.

`moby-runc` is a dependency for `moby-containerd` and is automatically installed with `moby-containerd`.

Reply:
Got it.

Should we invoke `removeMoby` at the beginning of the containerd installation process to ensure we don't ship both packages?

Likewise we should probably invoke `removeContainerd` during moby installation.